.gitignore

```diff
@@ -1,3 +1,5 @@
 *~
 cloudbuild.yaml
 cloudbuild_test.yaml
+mkdebootstrap/mkimage.sh
```

Comment:
mkimage needs to be checked in. anyone checking out the code needs a copy of it to build debian. the other two files are generated so they don't need to be checked in.